chaperone.json

```
@@ -0,0 +1,11 @@
+{
+    "app_id": "uhCkkmrkoAHPVf_eufG7eC5fm6QKrW5pPMoktvG5LOC0SnJ4vV1Uv",
+    "log_level": true,
+    "connection": {
+      "secure": false,
+      "address": "\"localhost\"",
+      "port": 42233,
+      "path": "/"
+    },
+    "mode": 2
+  }
```

```suggestion
  }

```

Not sure if you care about trailing newlines